index.js

```diff
@@ -130,7 +130,7 @@ module.exports = function (opts) {
     actionEntryPoint: endpoint('/actions/views/entry_point', Opts, Readable),
     aliasExecution: endpoint('/aliasexecution', Opts, Writable),
     executions: endpoint('/executions', Opts, Readable, Writable, Paginatable, Watchable,
-      Attributable, Repeatable),
+      Attributable, Repeatable, Deletable),
```

Comment:
I would prefer "cancelable" to be more explicit, but I know we use `DELETE` http method, so I guess that's OK :P